backend/notifier.py

```diff
@@ -144,26 +144,32 @@ def converter(d: Optional[datetime]) -> str:
                 return f'{d.month}/{d.day}'
             return f'{d.year}/{d.month}/{d.day}'
         prev, curr = converter(prev), converter(curr)
-        content = f"NEW {embassy.name_cn}: {prev} -> {curr}"
+        content = f"{embassy.name_cn}: {prev} -> {curr}"
         # qq
         extra = SECRET["qq"]
         base_uri = extra["mirai_base_uri"]
         auth_key = extra["mirai_auth_key"]
         qq_num = extra["qq_num"]
         if embassy.region == "DOMESTIC":
             group_id = extra["qq_group_id"]["domestic"]
-        else:
+        elif embassy.code in DEFAULT_FILTER:
             group_id = extra["qq_group_id"]["non_domestic"]
-        r = requests.post(base_uri + "/auth",
-                          data=json.dumps({"authKey": auth_key})).json()
-        session = r["session"]
-        requests.post(base_uri + "/verify",
-                      data=json.dumps({"sessionKey": session, "qq": qq_num}))
-        for g in group_id:
-            requests.post(base_uri + "/sendGroupMessage", data=json.dumps(
-                {"sessionKey": session, "target": g, "messageChain": [{"type": "Plain", "text": content}]}))
-        requests.post(base_uri + "/release",
-                      data=json.dumps({"sessionKey": session, "qq": qq_num}))
+        else:
+            group_id = []
+        if group_id:
+            r = requests.post(base_uri + "/auth",
+                              data=json.dumps({"authKey": auth_key})).json()
+            session = r["session"]
+            requests.post(base_uri + "/verify",
+                          data=json.dumps({"sessionKey": session, "qq": qq_num}))
```

Comment:
I've been curious about the right way for indenting a long function call, as I'm unsure whether the way I do it follow the coding convention. Would be much appreciated if you can refer me to the doc of this way of indentation.